google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -94,9 +116,7 @@ class Client(object):
         )
     """
 
-    _batch_class = thread.Batch
-
-    def __init__(self, batch_settings=(), **kwargs):
+    def __init__(self, publisher_options=(), batch_settings=(), **kwargs):
```

Comment:
This will be an issue, as `batch_settings` will now suddenly be an empty tuple, as the positional argument will be assigned to `publisher_options` instead.

Reply:
I had mistakenly thought that the parameter names were required. I fixed this in commit d688a81.